dbsettings/settings.py

```diff
@@ -5,4 +5,5 @@
 sites_installed = apps.is_installed('django.contrib.sites')
 USE_SITES = getattr(settings, 'DBSETTINGS_USE_SITES', sites_installed)
 USE_CACHE = getattr(settings, 'DBSETTINGS_USE_CACHE', True)
+CACHE_EXPIRATION = getattr(settings, 'DBSETTINGS_CACHE_EXPIRATION', 1*60*60)
```

Comment:
I'm not convinced to this change. It effectively overrides the default cache timeout (whatever it was), but also silently forces a timeout of 1*60*60 if nobody set anything (that is not backward-compatible; if someone upgrades dbsettings to newer version with this change, timeout suddenly changes).

I'd rather see the default value here as -1 (None is also a valid cache timeout meaning "Infinity"), and in the code do:
```
args = []
if CACHE_EXPIRATION != -1:
  args.append(CACHE_EXPIRATION)
cache.set(key, storage, *args)
```
or something similar with kwargs

Reply:
Also, regarding `so the items will not stay in the cache forever` - have you tried playing with TIMEOUT (https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-CACHES-TIMEOUT) ? By default they should expire after 5 minutes, not stay forever.

Reply:
Thanks for the quick reply. I changed the code according to your comments.

Reply:
Thanks for the PR and fixing the exceptions problem.
Probably tomorrow I'll test it, document and release.

Reply:
Hi again @zlorf. I don't want to make you uncomfortable but we would greatly appreciate it if you could finish it as soon as possible.